src/task/toggle-enabled-for-item-validation-process-task.ts

```diff
@@ -0,0 +1,39 @@
+// global
+import { DatabaseTransactionHandler, Member } from 'graasp';
+// local
+import { ItemValidationService } from '../db-service';
+import { BaseValidationTask } from './base-validation-task';
+import { ItemValidationProcess } from '../types';
+
+type InputType = { id: string; enabled: boolean };
```

Comment:
I think you reuse it in the task manager or service-api, so export this type and reuse it.

Reply:
The other input type doesn't have `id` field. I think this is only used here so I'll just leave it. The other places I already created a type for that